### PR TITLE
Fix mission placeholder

### PR DESCRIPTION
**Pull request by Eddasol**

### CODE REVIEW

File: frontend/src/components/Pages/FrontPage/MissionOverview/MissionQueueView.tsx [hunk 1]
```diff
@@ -134,11 +135,15 @@ export function MissionQueueView() {
         })
     }, [missionQueue, ongoingMissions])
 
-    const missionQueueDisplay = missionQueue
-        .filter((m) => m.installationCode?.toLocaleLowerCase() === installationCode.toLocaleLowerCase())
-        .map((mission, index) => (
-            <MissionQueueCard key={index} order={index + 1} mission={mission} onDeleteMission={onDeleteMission} />
-        ))
+    useEffect(() => {
+        setMissionsToDisplay(
```
Comment (eivindsjovold) [inline frontend/src/components/Pages/FrontPage/MissionOverview/MissionQueueView.tsx:139]: could this (repeated) code be written into a module?

File: frontend/src/components/Pages/FrontPage/MissionOverview/OngoingMissionView.tsx [hunk 2]
```diff
@@ -34,18 +36,26 @@ export function OngoingMissionView() {
     const { TranslateText } = useLanguageContext()
     const { ongoingMissions } = useMissionsContext()
     const { installationCode } = useInstallationContext()
+    const [missionsToDisplay, setMissionsToDisplay] = useState<Mission[]>([])
```
Comment (mrica-equinor) [inline frontend/src/components/Pages/FrontPage/MissionOverview/OngoingMissionView.tsx:39]: I think I would like to call this ongoingMissionsToDisplay but also understand if you think it's too big of a name and not justifiable

**Review by mrica-equinor — APPROVED** [review]
LGTM :) check the comment but it is working well

File: frontend/src/components/Pages/FrontPage/MissionOverview/MissionQueueView.tsx (hunk 1 above)
Comment (Eddasol) [reply frontend/src/components/Pages/FrontPage/MissionOverview/MissionQueueView.tsx:139]: Is it worth making a module for so few lines?

Comment (eivindsjovold) [reply frontend/src/components/Pages/FrontPage/MissionOverview/MissionQueueView.tsx:139]: Not sure - if it is not a lot of extra effort i would say it is

**Review by eivindsjovold — APPROVED** [review]
LGTM

**Pull request merged**